app/pkg/cputemp/cputemp.go

```diff
@@ -36,10 +36,16 @@ func getCPUTemp() CPUTemp {
 }
 
 func saveCPUTemp(pathToSave string, cpuTemp CPUTemp) {
-	f, _ := os.OpenFile(fmt.Sprintf("%s/cputemp.log", pathToSave), os.O_APPEND|os.O_WRONLY|os.O_CREATE, 0666)
+	f, err := os.OpenFile(fmt.Sprintf("%s/cputemp.log", pathToSave), os.O_APPEND|os.O_WRONLY|os.O_CREATE, 0666)
+	if err != nil {
+		fmt.Printf("Could not open file %q: %v", fmt.Sprintf("%s/cputemp.log", pathToSave), err)
```

Comment:
I think you should return the error (enriched if necessary). 
In this way the caller of `saveCPUTem` can handle the error.